test/tstKDOP.cpp

```diff
@@ -140,17 +140,17 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(intersects_point_kdop, KDOP_t, KDOP_3D_types)
   {
     KDOP_t x; // unit cube with (1,1,0)--(1,1,1) edge chopped away
     // bottom face
-    expand(x, Point(0, 0, 0));
-    expand(x, Point(1, 0, 0));
-    expand(x, Point(1, 0.75, 0));
-    expand(x, Point(0.75, 1, 0));
-    expand(x, Point(0, 1, 0));
+    expand(x, Point{0, 0, 0});
+    expand(x, Point{1, 0, 0});
+    expand(x, Point{1, 0.75, 0});
+    expand(x, Point{0.75, 1, 0});
+    expand(x, Point{0, 1, 0});
```

Comment:
What the reason for changing here?

Reply:
Hyper point does not have a constructor, only aggregate initialization.